src/js/utils/option-utils.jsx

```diff
@@ -218,3 +219,8 @@ export const fetchProductsTags = async () => {
   const response = await apiClient.get('/openboxes/api/tagOptions');
   return response.data.data;
 };
+
+export const fetchProductsGlAccounts = async () => {
+  const { data } = await glAccountApi.getGlAccountOptions();
+  return data.data;
+};
```

Comment:
I'm adding one additional comment, which should be rather answered by @awalkowiak - I'm wondering if we should wrap it inside `try/catch` and have somewhat user-friendly error message if any happens, or let the handler handle that, but we could have some "ugly" error message in terms of user perspective?

Reply:
Discussion for standardizing error messages #3849